__tests__/app.test.js

```diff
@@ -47,4 +47,49 @@ describe("/api/topics", () => {
       });
   });
 
+})
+
+describe("/api/article/:article_id", () => {
+  test("GET:200 sends an the article object with articel_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+      const date = new Date('2020-07-09 21:11:00') 
+        expect(response.body.article.length).toBe(1);
+        expect(response.body.article[0]).toEqual(expect.objectContaining({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: date.toISOString(),
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        }));
+        
+        response.body.article.forEach((article) => {
```

Comment:
This is not necessary at all as you are asserting the data types in the test above.